flow/render_status.py

```diff
@@ -0,0 +1,220 @@
+        if template is None:
+            if detailed and expand:
+                template = 'status_expand.jinja'
+            elif detailed and not unroll:
+                template = 'status_stack.jinja'
+            elif detailed and compact:
+                template = 'status_compact.jinja'
+            else:
+                template = 'status.jinja'
+
+        def draw_progressbar(value, total, escape='', width=40):
+            """Visualize progess with a progress bar.
+
+            :param value:
+                The current progress as a fraction of total.
+            :type value:
+                int
+            :param total:
+                The maximum value that 'value' may obtain.
+            :type total:
+                int
+            :param width:
+                The character width of the drawn progress bar.
+            :type width:
+                int
+            """
+
+            assert value >= 0 and total > 0
+            ratio = ' %0.2f%%' % (100 * value / total)
+            n = int(value / total * width)
+            return escape + '|' + ''.join(['#'] * n) + ''.join(['-'] * (width - n)) \
+                          + escape + '|' + ratio
+
+        def job_filter(job_op, scheduler_status_code, all_ops):
+            """filter eligible jobs for status print.
+
+            :param job_ops:
+                Operations information for a job.
+            :type job_ops:
+                OrderedDict
+            :param scheduler_status_code:
+                Dictionary information for status code
+            :type scheduler_status_code:
+                dict
+            :param all_ops:
+                Boolean value indicate if all operations should be displayed
+            :type all_ops:
+                bool
+            """
+
+            if scheduler_status_code[job_op['scheduler_status']] != 'U' or \
+               job_op['eligible'] or all_ops:
+                return True
+            else:
+                return False
+
+        def get_operation_status(operation_info, symbols):
+            """Determine the status of an operation.
+
+            :param operation_info:
+                Dictionary containing operation information
+            :type operation_info:
+                dict
+            :param symbols:
+                Dictionary containing code for different job status
+            :type symbols:
+                dict
+            """
+
+            if operation_info['scheduler_status'] >= JobStatus.active:
+                op_status = 'running'
+            elif operation_info['scheduler_status'] > JobStatus.inactive:
+                op_status = 'active'
+            elif operation_info['completed']:
+                op_status = 'completed'
+            elif operation_info['eligible']:
+                op_status = 'eligible'
+            else:
+                op_status = 'ineligible'
+
+            return symbols[op_status]
+
+        if pretty:
```

Comment:
This block can be simplified. We definitely need to remove the unused args from the docstring. Furthermore, you might be able to get `pretty` from the global scope and not conditionally define the function twice:
```python
def highlight(s, eligible):
    # insert docstring here
    if eligible and pretty:
        return '**' + s + '**'
    else:
        return s
```
or even
```python
highlight = lambda s, eligible: '**' + s + '**' if pretty and eligible else s
```
but using a lambda loses the documentation so that wouldn't be my preference.